paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/samples/ui/wallet/SavedPaymentMethodsViewModel.kt

```diff
@@ -28,51 +30,75 @@ class SavedPaymentMethodsViewModel(
     )
     val state: StateFlow<SavedPaymentMethodsViewState> = _state
 
-    init {
-        initialize()
-    }
+    internal val customerAdapter: CustomerAdapter = CustomerAdapter.create(
+        context = getApplication(),
+        customerEphemeralKeyProvider = {
+            fetchCustomerEphemeralKey().fold(
+                success = {
+                    kotlin.Result.success(
```

Comment:
Hmmm... This really sucks. What's going on here?

Reply:
Fuel has their own Result class

Reply:
Maybe import alias fuel result? `import com.github.kittinunf.result.Result as FuelResult`